src/pages/gateway/handlers/openapi.md

```diff
@@ -53,7 +53,7 @@ The OpenAPI handler uses the following naming conventions:
 
 The `operationId` is only modified when necessary according to the GraphQL spec:
 
-  - The following characters are replaced with an underscore (`_`): white space, `.`, `/`, `:` and `-`.
+  - The following characters are replaced with an underscore (`_`): whitespace, `.`, `/`, `:` and `-`.
```

Comment:
If we're relying on MS, this should be two words: https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/w/white-space

Reply:
According to the Adobe Glossary should be two words

Reply:
I updated the textlint's rule configuration